src/lib/multiaddres.ts

```diff
@@ -288,7 +294,8 @@ export async function resolveRelay({
   ...args
 }: { maybeRelay: string | undefined } & ResolveNodesArgs) {
   return (
-    (await getMaybeNamedNode(maybeRelay, args.maybeFluenceConfig))?.multiaddr ??
+    (await getMaybeNamedAddrAndPeerId(maybeRelay, args.maybeFluenceConfig))
+      ?.multiaddr ??
     maybeRelay ??
     (await getRandomRelayAddr(args))
   );
```

Comment:
this is super hard to follow. Two ternary operators is too much. Especially with inlined functions.

I'd suggest to move all the function calls to separate variables, and then maybe use `if`s

Reply:
ok. will refactor. but thease are not ternaries